app/controllers/miq_policy_controller.rb

```diff
@@ -863,10 +870,12 @@ def handle_selection_buttons(members,
         end
         params[members_chosen].each do |mc|
           idx = nil
-          @edit[:new][members].each_with_index { |mem, i| idx = mem[-1] == mc.to_i ? i : idx }  # Find the index in the new members array
+          # Find the index in the new members array
+          @edit[:new][members].each_with_index { |mem, i| idx = mem[-1] == mc.to_i ? i : idx }
```

Comment:
Not for this PR, but the copy / paste of this line (and parent block) is killing me.